Explorer/Assets/DCL/AvatarRendering/AvatarShape/Rendering/TextureArray/TextureArrayHandler.cs

```diff
@@ -93,10 +99,16 @@ private TextureArraySlotHandler CreateHandler(Vector2Int resolution)
             handlersByResolution[resolution] = slotHandler;
 
             // When the handler is created initialize the default texture
-            if (defaultTextures.TryGetValue(new TextureArrayKey(textureID, resolution), out var defaultTexture))
+            if (defaultTextures.TryGetValue(new TextureArrayKey(textureID, resolution, 0), out var defaultTexture0))
```

Comment:
You can do a `for` with the size of `defaultTextures` to make it scalable

Reply:
done